staging/src/k8s.io/cri-client/pkg/remote_runtime.go

```diff
@@ -107,6 +108,7 @@ func NewRemoteRuntimeService(endpoint string, connectionTimeout time.Duration, t
 			grpc.WithUnaryInterceptor(otelgrpc.UnaryClientInterceptor(tracingOpts...)),
 			grpc.WithStreamInterceptor(otelgrpc.StreamClientInterceptor(tracingOpts...)))
 	}
+	dialOpts = append(dialOpts, grpc.WithMaxMsgSize(math.MaxInt64))
```

Comment:
That is an absurd size.

the problem in your environment is failure to delete pods and should be fixed by fixing the container runtime

Reply:
agree with @BenTheElder sorry no.

/hold